lms/djangoapps/grades/new/course_grade.py

```diff
@@ -274,13 +333,26 @@ def _get_saved_grade(self, course, course_structure):  # pylint: disable=unused-
         """
         Returns the saved grade for the given course and student.
         """
-        if PersistentGradesEnabledFlag.feature_enabled(course.id):
-            # TODO LATER Retrieve the saved grade for the course, if it exists.
-            _pretend_to_save_course_grades()
+        if not PersistentGradesEnabledFlag.feature_enabled(course.id):
+            return None
 
+        current_grading_policy_hash = course_structure.get_transformer_block_field(
+            course.location,
+            GradesTransformer,
+            'grading_policy_hash'
+        )
```

Comment:
Nit: Consider moving the call to get the current grading_policy_hash into the `load_persisted_grade` method - so that logic is consolidated into the `CoruseGrade` class.  You can then also DRY it by having an internal helper to get that value given a `course_structure`.